tap/tap.py

```diff
@@ -226,23 +226,23 @@ def _add_argument(self, *name_or_flags, **kwargs) -> None:
                     loop = False
                     types = get_args(var_type)
 
-                    # Don't allow Tuple[()]
-                    if len(types) == 1 and types[0] == tuple():
-                        raise ArgumentTypeError('Empty Tuples (i.e. Tuple[()]) are not a valid Tap type '
-                                                'because they have no arguments.')
-
                     # Handle Tuple[type, ...]
                     if len(types) == 2 and types[1] == Ellipsis:
                         types = types[0:1]
                         loop = True
                         kwargs['nargs'] = '*'
+                    # Handle Tuple[()]
```

Comment:
In Python 3.11, `Tuple[()]` is treated the same as `Tuple` (previously it was treated differently) and is now handled by (new) line 244 since `get_args` now has length 0 instead of 1.